addons/hr_holidays/static/src/owl/components/thread/thread.xml

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<templates xml:space="preserve">
+    <t t-inherit="mail.component.Thread" t-inherit-mode="extension">
+        <xpath expr="//t[@name='loading_condition']" position="before">
+            <t t-if="storeProps.directPartner and storeProps.directPartner.out_of_office_date_end">
+                <div class="o_Thread_outOfOffice alert alert-primary my-0" t-esc="getOutOfOfficeText()" role="alert"/>
```

Comment:
not fond of classname whose name and intent are solely styling stuffs, I prefer adding a style rule in CSS for `.o_Thread_outOfOffice` instead

Reply:
Ok for `my-0` but I'm not rewriting the css for `alert`